frontend/build-configs/bayern/index.ts

```diff
@@ -1,15 +1,31 @@
-import BuildConfigType from "../types";
+import BuildConfigType, {CommonBuildConfigType} from "../types";
 
-export const bayernCommon = {
+export const bayernCommon: CommonBuildConfigType = {
     appName: "Ehrenamt",
     appIcon: "",
-    backendUrl: "",
+    projectId: "bayern.ehrenamtskarte.app",
+    categories: [1, 2, 3],
+    theme: {
+        primaryColor: "#123456",
+        secondaryColor: "#123456"
+    },
+    mapDataHost: {
+        staging: "https://tiles.staging.bayern.entitlementcard.app",
+        production: "https://tiles.bayern.entitlementcard.app",
+        local: "http://localhost:5002",
+    },
+    backendUrl: {
+        staging: "https://api.staging.bayern.entitlementcard.app",
+        production: "https://api.bayern.entitlementcard.app",
+        local: "http://10.0.2.2:7000",
```

Comment:
Should we use 10.0.2.2 also for mapDataHost.local?

Reply:
I'm not sure I tried to keep it like it was. I think all of this needs probably proper values in the future. Staging and production are probably not working right now.